test/bin/ci_phase_iso_boot.sh

```diff
@@ -57,13 +57,16 @@ fi
 # Tell scenario.sh to merge stderr into stdout
 export SCENARIO_MERGE_OUTPUT_STREAMS=true
 
+# Prepare all the scenarios that need to run into a special directory
+prepare_scenario_sources
```

Comment:
This is going to redo the scenario file setup. Why do we need to do that?

Reply:
Both boot and test scripts glob the scenario files based on directories. For this change we need to exclude a specific file out of those, so I thought about preparing a special directory where all tests that should run are included. This allows further filtering, not just globing an entire dir.

Reply:
You're doing the work twice. Why does it need to be done in both the build and boot phases?

Reply:
Yes, because we want to avoid booting CNCF dedicated 2xVMs and also running tests on them.